ztp/source-crs-builder/MachineConfigAcceleratedStartup/accelerated-container-startup.service

```diff
@@ -0,0 +1,31 @@
+[Unit]
+Description=Unlocks more CPUs for critical system processes during container startup
+
+[Service]
+Type=simple
+ExecStart=/usr/local/bin/accelerated-container-startup.sh
+
+# Maximum wait time is 600s = 10m:
+Environment=MAXIMUM_WAIT_TIME=600
+
+# Steady-state threshold = 2%
+# Allowed values:
+#  4  - absolute pod count (+/-)
+#  4% - percent change (+/-)
+#  -1 - disable the steady-state check
+Environment=STEADY_STATE_THRESHOLD=2%%
```

Comment:
Yes, in systemd units you need to escape `%` because it's used for "specifiers"

See "table 4" in https://www.freedesktop.org/software/systemd/man/systemd.unit.html

Reply:
Added a comment to the service file to make this more obvious.